dns/coredns.go

```diff
@@ -123,6 +125,11 @@ func (c *Mgr) AddSubdomain(user string, domain string, IPs []net.IP) error {
 		return err
 	}
 
+	// if this is a managed domain and there is already some records, then refuse to modify it
+	if owner.Owner == c.identity && len(zr.Records) > 0 {
+		return fmt.Errorf("the sub-domain %s is already used by someone else: %w", domain, ErrAuth)
```

Comment:
this means we can not change a subdomain IP even if we made it?

Reply:
Well that's a bit the trick I used here. I do not know who have create the subdomain...
The only thing I know is this subdomain already exists.